### DISCUSSION TITLE
How to build and run test code

### DISCUSSION THREAD

**Discussion by za** [aquasecurity/trivy#5254] [Development]
Hi there, 

is it possible to have a documentation on _how to build and run trivy test code_? I was looking at this on docs but I couldn't find it (or probably I missed it?).

Maybe it's a good start to get more developers involved.

**Comment by knqyf263** [comment]
https://aquasecurity.github.io/trivy/v0.45/community/contribute/pr/

**Comment by za** [comment]
Ah, yes I missed it. It was part of PR section. I thought it was about creating PR _only_. 

Thanks @knqyf263 .
